tests/components/risco/test_sensor.py

```diff
@@ -188,14 +190,13 @@ async def test_setup(hass, two_zone_alarm):  # noqa: F811
     for category, entity_id in ENTITY_IDS.items():
         _check_state(hass, category, entity_id)
```

Comment:
The test fails already here. So we probably need to wait until the second state update by time fast forward until we check state.

Reply:
I see. I think the right way to solve this is to make sure the coordinator doesn't run until the binary sensor platform is loaded (otherwise it can happen in production as well). I'm not explicitly running it, so I'm not sure what the trigger is and how to prevent it.

Reply:
maybe I can add the listeners on the homeassistant start even instead of `async_added_to_hass`?

Reply:
Sorry I actually am calling `await self.coordinator.async_request_refresh()` in `async_added_to_hass`.
However, if I remove that, the coordinator isn't called, even after moving time forward.

Reply:
OK, I think I solved it